docs/RELEASE-NOTES.rst

```diff
@@ -6,15 +6,17 @@ next release
 Added Functionality
 `````````````````````
```

Comment:
1.10 ?

Reply:
That will be updated as part of release process.